opentelemetry-user-events-logs/examples/README.md

```diff
@@ -0,0 +1,100 @@
+# Getting Started with OpenTelemetry Rust Logs in 5 Minutes - Exporting logs to user_events
+
+## Prerequisites
+
+* Rust and Cargo: Install them if you haven't.
+* A Linux system: Make sure it supports user_events.
+* Correct Permissions: Linux user(preferable non-root) with write permission to the tracefs.
+* A listener/agent to listen for new events. Or else `perf` and `decode-perf` tools installed for validation.
```

Comment:
Provide official links to these prerequisites.

Reply:
Have added them under the useful links section at the end. Will put it here instead.